src/App.tsx

```diff
@@ -1,10 +1,40 @@
-import React from 'react';
+import {
+  Navigate,
+  Route,
+  Routes,
+} from 'react-router-dom';
 
 import './App.scss';
+import { Navigation } from './components/Navigation';
+import { HomePage } from './components/HomePage';
+import { PeopleList } from './components/PeopleList';
+import { NotFoundPage } from './components/NotFoundPage';
 
 const App = () => (
-  <div className="App">
-    <h1>People table</h1>
+  <div className="container is-max-desktop">
+    <div>
+      <Navigation />
+    </div>
+    <section className="section">
+      <div className="container">
+        <div className="columns">
+          <div className="column">
+            <Routes>
+              <Route path="/" element={<HomePage />} />
+              <Route path="/people" element={<PeopleList />} />
+              <Route
+                path="*"
+                element={<NotFoundPage />}
+              />
+              <Route
+                path="/home"
+                element={<Navigate to="/" />}
+              />
```

Comment:
You can avoid `/` in each Route with this syntax 

https://reactrouter.com/docs/en/v6/getting-started/overview#nested-routes

<img width="459" alt="Screenshot 2022-03-17 at 12 53 44" src="https://user-images.githubusercontent.com/37452763/158794214-4eb50a98-de03-4c86-945b-ac4fe27184e1.png">